e2e_tests/tests/per_provider/normal_tests/hash_calib.rs

```diff
@@ -0,0 +1,64 @@
+// Copyright 2020 Contributors to the Parsec project.
```

Comment:
Know that this file won't get executed if the module is not put in `e2e_tests/tests/per_provider/normal_tests/mod.rs` 😋 Also if those are the tests that pass on the crypto auth lib provider, I think it's still worth putting them inside the `hash.rs` file so that all providers benefit from this testing.